README.rst

```diff
@@ -3,6 +3,11 @@ E-Commerce plugin for `Tutor <https://docs.tutor.edly.io>`_
 
 This is a plugin for `Tutor <https://docs.tutor.edly.io>`_ that integrates the `E-Commerce <https://github.com/openedx/ecommerce/>`__ application in an Open edX platform.
 
+⛔️ WARNING
+==========
+
+This service is under-maintained. We are not fixing bugs or developing new features for it. We hope to deprecate and replace it soon. For updates, `follow along on the DEPR ticket <https://github.com/openedx/public-engineering/issues/22>`_.
+
```

Comment:
nit: Instead of "This service", "We", the wording should be renamed to signify:
-  This service --> e-commerce and e-commerce worker
- We --> Axim